apps/laboratory/tests/email.spec.ts

```diff
@@ -17,8 +17,8 @@ const emailTest = test.extend<{ library: string }>({
 
 emailTest.describe.configure({ mode: 'serial' })
 
-emailTest.beforeAll(async ({ browser, library }, testInfo) => {
-  emailTest.setTimeout(120000)
+emailTest.beforeAll(async ({ browser, library }) => {
+  emailTest.setTimeout(180000)
```

Comment:
Allotting more time for email delivery

Reply:
why is this required? were emails not arriving in 2 mins?